quipucords/api/management/commands/regenerate_missing_report_cache_files.py

```diff
@@ -71,16 +74,10 @@ def generate_fingerprints(
         """Generate cached fingerprints for the given deployments report IDs."""
         success_ids = []
         failure_ids = []
-        deployments_report_ids = sorted(list(deployments_report_ids))
-        for _id in deployments_report_ids:
-            deployments_report = DeploymentsReport.objects.get(pk=_id)
-            self.write_info(f"Rerunning fingerprint task for {deployments_report}")
-            try:
-                deployments_report._rerun_latest_fingerprint(wait=True)
-                success_ids.append(_id)
-            except Exception as e:  # noqa: BLE001
-                self.stdout.write(self.style.ERROR(str(e)))
-                failure_ids.append(_id)
+        for _id in sorted(list(deployments_report_ids)):
+            success_ids.append(_id) if generate_cached_fingerprints(
+                _id
+            ) else failure_ids.append(_id)
```

Comment:
This formatting turned out ugly. Can we follow the same pattern as in `generate_csvs`?
```suggestion
            success = generate_cached_fingerprints(_id)
            success_ids.append(_id) if success else failure_ids.append(_id)
```

Reply:
Agreed! Make consistent in latest push.